staging/src/k8s.io/apiserver/pkg/storage/cacher/time_budget.go

```diff
@@ -19,11 +19,14 @@ package cacher
 import (
 	"sync"
 	"time"
+
+	"k8s.io/apimachinery/pkg/util/clock"
 )
 
 const (
-	refreshPerSecond = 50 * time.Millisecond
-	maxBudget        = 100 * time.Millisecond
+	refreshPerInterval = 50 * time.Millisecond
+	refreshInterval    = 1 * time.Second
```

Comment:
I suggest simplifying a bit and just changing to just having "refreshPerSecond".

refreshInternal is just a multiplication factor for that, so you can achieve the same by having just refreshPerSecond.

Same below in timeBudgetImpl.